SignUpActivity.java

```diff
@@ -0,0 +1,137 @@
+package com.manoj.androidclub;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import androidx.appcompat.app.AppCompatActivity;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import java.util.Calendar;
+
+public class SignUpActivity extends AppCompatActivity {
+
+    DatabaseReference mDatabase;
+    FirebaseUser mCurrentUser;
+    String mEmail;
+    String mName;
+    String mDepartmant;
+    String mYear;
+    String mRegNumber;
+    String mCollege;
+    String mPassword;
+    String mConfirmPassword;
```

Comment:
no issues for now but conventionally it should be uEmail,uName  and so on and instances should be start with mName